dotnet/README.MD

```diff
@@ -0,0 +1,179 @@
+ 
+ ### Enable the Cloud Bigtable APIs.
+ 
+ Make sure to [enable the Bigtable APIs][enable-bigtable-api].
+ 
+ [enable-bigtable-api]: https://console.cloud.google.com/apis/library?q=bigtable
+ 
+ ### Install the Google Cloud SDK.
+ 
+ If you haven't already installed the Google Cloud SDK, [install the Google
+ Cloud SDK][cloud-sdk] now. The SDK contains tools and libraries that enable you
+ to create and manage resources on Google Cloud Platform.
+ 
+ [cloud-sdk]: https://cloud.google.com/sdk/
+ 
+ ### Setting Google Application Default Credentials
+ 
+ Set your [Google Application Default
+ Credentials][application-default-credentials] by [initializing the Google Cloud
+ SDK][cloud-sdk-init] with the command:
+ 
+     gcloud init
+ 
+ Generate a credentials file by running the [application-default login](https://cloud.google.com/sdk/gcloud/reference/auth/application-default/login) command:
+ 
+     gcloud auth application-default login
+ 
+ [cloud-sdk-init]: https://cloud.google.com/sdk/docs/initializing
+ [application-default-credentials]: https://developers.google.com/identity/protocols/application-default-credentials
+ 
+ 
+ ## Provisioning an instance
+ 
+ Follow the instructions in the [user
+ documentation](https://cloud.google.com/bigtable/docs/creating-instance) to
+ create a Google Cloud Platform project and Cloud Bigtable instance if necessary.
+ You'll need to reference your project id and instance id to run the
+ application.
+ 
+ 
+ ## Running the application
+ 
+ In appsettings.json replace `[PROJECT_ID]` with google cloud project name and `[BIGTABLEINSTANCE]` with what you called it when provisioning the instance.
+ 
+    "projectId": `[PROJECT_ID]`,
+    "instanceId": `[BIGTABLEINSTANCE]`,
+ 
+ Run the sample using `dotnet run` CLI command
+    
+    dotnet run -c Release
+ 
+ You will see output resembling the following, interspersed with informational logging
+ from the underlying libraries:
+ 
+    Creating new table: Hello-Bigtable
+    Table Hello-Bigtable created succsessfully
+
+    Writing some greetings to the table Hello-Bigtable
+    Reading the first row
+            Row key: greeting0, Value: Hello World!
+    Reading all rows using streaming
+            Row key: greeting0, Value: Hello World!
+            Row key: greeting1, Value: Hellow Bigtable!
+            Row key: greeting2, Value: Hellow C#!
+    Deleting table: Hello-Bigtable
+    Table Hello-Bigtable deleted succsessfully
+    Press any key to exit
+ 
+ 
+ ## Cleaning up
+ 
+ To avoid incurring extra charges to your Google Cloud Platform account, remove
+ the resources created for this sample.
+ 
+ 1.  Go to the Clusters page in the [Cloud
+     Console](https://console.cloud.google.com).
+ 
+     [Go to the Clusters page](https://console.cloud.google.com/project/_/bigtable/clusters)
+ 
+ 2.  Click the cluster name.
+ 
+ 3.  Click **Delete**.
+ 
+     ![Delete](https://cloud.google.com/bigtable/img/delete-quickstart-cluster.png)
```

Comment:
That file needs to be removed, please use https://cloud.google.com/bigtable/img/delete-quickstart-instance.png instead.

Reply:
How did you find this image and these instructions BTW? Are we doing this in another example in this repo? Please file an issue; we should update other instances of this as well.

Reply:
I see, this is the style of example we have everywhere, which needs to be updated.

I filed https://github.com/GoogleCloudPlatform/cloud-bigtable-examples/issues/274 and fixed this in https://github.com/GoogleCloudPlatform/cloud-bigtable-examples/pull/275 — please use the same format as done in that PR.

I will file a separate issue to factor out the section about cleaning up resources, so that we don't have to keep track of 5-6 separate copies of the same steps.

Reply:
I filed issue https://github.com/GoogleCloudPlatform/cloud-bigtable-examples/issues/276 to factor out the common instructions such as deleting an instance into a separate place we can link to; for now, please feel free to update these instructions from the other version, and we can refactor this later (i.e., don't wait for the new common cleanup file to be created first before finishing this PR).